server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -815,6 +854,7 @@ protected void doRun() throws Exception {
                         rootBlobs,
                         repositoryData,
                         repositoryMetaVersion,
+                        null, // Passing null since no remote store lock files need to be cleaned up.
```

Comment:
any reason to not call the new method `deleteSnapshotsAndReleaseLockFiles` with null `remoteStoreLockManagerFactory` from this method ?

Reply:
We can do that! Made the change in revision.